src/pages/FavoritesPage/FavoritesPage.js

```diff
@@ -0,0 +1,66 @@
+import { useHistory } from "react-router-dom";
+import PropTypes from "prop-types";
+import styled from "styled-components/macro";
+import Card from "../../components/Card/Card";
+import Header from "../../components/Header/Header";
+
+FavoritesPage.propTypes = {
+  history: PropTypes.func.isRequired,
+  handleToggleIsFavorite: PropTypes.func,
+};
+
+export default function FavoritesPage({ favorites, handleToggleIsFavorite }) {
+  let history = useHistory();
+
+  return (
+    <>
+      <Wrapper>
+        <Header>Your favorites</Header>
+        <div>
+          <BackButton onClick={() => history.goBack()}>&lt; back</BackButton>
+        </div>
+        <ListWrapper>
+          {favorites.map(({ id, image, title, text, label, isFavorite }) => {
+            return (
+              <Card
+                key={id}
+                image={image}
+                title={title}
+                text={text}
+                label={label}
+                isFavorite={isFavorite}
+                onToggleIsFavorite={() => handleToggleIsFavorite(id)}
```

Comment:
I think you dont need the callbback here but I am not sure
```suggestion
                onToggleIsFavorite={handleToggleIsFavorite}
```